semantic/dates.py

```diff
@@ -227,7 +227,7 @@ def safe(exp):
                 d = self.now
             elif tomorrow:
                 d = self.now + datetime.timedelta(days=1)
-            elif day_of_week:
+            elif day_of_week is not None:
```

Comment:
Thanks for this. I don't think this will properly handle the case in which we fall-through from `safe`, and thus `day_of_week` is `False`. Maybe `type(day_of_week) == int`?